src/platforms/node/guides/aws-lambda/index.mdx

```diff
@@ -26,7 +26,7 @@ npm install --save @sentry/serverless
 yarn add @sentry/serverless
 ```
 
-We also support [installing Sentry in Lambda Layer](/platforms/node/guides/aws-lambda/layer/).
+We also support [installing Sentry in Lambda Layer](/platforms/node/guides/aws-lambda/layer/) and [installing Sentry as a Container Image](/platforms/node/guides/aws-lambda/container-image/).
```

Comment:
Switching the order of these to match the order in which they're listed in the navigation

```suggestion
We also support [installing Sentry as a Container Image](/platforms/node/guides/aws-lambda/container-image/) and [installing Sentry in Lambda Layer](/platforms/node/guides/aws-lambda/layer/).
```